lib/active_merchant/net_http_ssl_connection.rb

```diff
@@ -1,11 +1,46 @@
 require 'net/http'
 
 module NetHttpSslConnection
+  module InnocuousCapitalize
+    def capitalize(name) = name
+    private :capitalize
+  end
+
+  class CaseSensitivePost < Net::HTTP::Post; prepend InnocuousCapitalize; end
```

Comment:
Is the `prepend` intended to give precedence to the existing behavior, which is capitalizing header fields?

Reply:
reponses:

1) no because of order, but because of classes see => hash_object.merge!(case_sensitve_header_hash), en-up being a plain ruby `Hash`, but case_sensitve_header_hash.merge!(hash_object) updates `CaseSensitiveHeaders`  instant but is still that class, so I can differentiate in the Refinement.

2) yes exactly that